packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -623,4 +624,18 @@ public void hideAndroidSoftKeyboard() {
             }
         }
     }
+
+    @ReactMethod
+    public void logException(final ReadableMap exception, final Callback jsCallback) {
+        LogExceptionCallback logExceptionCallback = onLogExceptionCallback(jsCallback);
+        mGutenbergBridgeJS2Parent.logException(exception, logExceptionCallback);
+    }
+
+   private LogExceptionCallback onLogExceptionCallback(final Callback jsCallback) {
```

Comment:
I'm following the same encapsulation used for other callbacks in the bridge, such as `ConnectionStatusCallback`.